pkg/operation/seed/seed.go

```diff
@@ -1440,7 +1461,7 @@ func cleanupOrphanExposureClassHandlerResources(ctx context.Context, c client.Cl
 		if exposureClassHandlerExists {
 			continue
 		}
-		log.Infof("Namespace %q is orphan as there is no ExposureClass handler in the gardenlet configuration anymore", namespace.Name)
+		log.Info("Namespace is orphan as there is no ExposureClass handler in the gardenlet configuration anymore")
```

Comment:
Isn't it a good idea to mention the namespace in the log as it was done earlier?

Reply:
The logger injected in this function already has this namespace value from [here](https://github.com/gardener/gardener/pull/6278/files/4e89c020d55ba43ee5d52ccba655cdd98be0e8cd#diff-8495b57b25d39744fe1b20b1e1ce695d130f730142af86ff9c5929afcdbabeb4R1452)